### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by GreysonElkins** [DougieDev/refactor-fitlit-B#38]
- [ ] `id` attributes on the page are not unique
- [ ] `<html>` element does not have a `lang` attribute
- [ ] `img` elements do not have `alt` attributes
![image](https://user-images.githubusercontent.com/62047446/87866373-44f0ab80-c93e-11ea-8aed-708164498bb0.png)

Passed audits:
* background and foreground colors have a sufficient contrast ratio
* document has a `<title>`
* lists contain only `<li>` elements and list supporting elements (`script` and `template`)
* list items are contained within `<ul>` parent elements

Many audits are currently n/a (including `aria`)
